addons/MultiAudio/src/presenter.js

```diff
@@ -295,13 +296,32 @@ function AddonMultiAudio_create(){
                         presenter.stop();
                         presenter.draggableItems[itemID].removeClass('playing');
                     }
+                },
+                drag : function(event, ui) {
+                    ui.position.left = ui.position.left / scale.X;
+                    ui.position.top = ui.position.top / scale.Y;
                 }
             });
 
             presenter.globalView.find(".wrapper-addon-audio").append($el);
             presenter.draggableItems[itemID] = $el;
     };
 
+    function getScale() {
+        var $content = $("#content");
+        if($content.size() > 0) {
+            var contentElem = $content[0];
+            var scaleX = contentElem.getBoundingClientRect().width / contentElem.offsetWidth;
+            var scaleY = contentElem.getBoundingClientRect().height / contentElem.offsetHeight;
```

Comment:
ContentElem.offsetWidth i Height nie mogą nigdy wynosić 0? Jeśli nie, to powinno być wszystko w porządku, a jeśli tak, to warto by dać tu jakieś zabezpieczenie, żeby nie wywaliło aplikacji.

Reply:
@rwojdak 
Nie powinno być zerem bo offset to width + padding + border, czyli zerem byłoby, kiedy element ma wymiar 0 px, co uniemożliwia draggowanie go. A poza tym to jest kawałek kodu, który działa w kilku innych miejscach w kodzie w takiej samej postaci. Zostawiłbym jak jest.